src/test/bridges/yearn/YearnUnit.t.sol

```diff
@@ -62,7 +62,7 @@ contract YearnBridgeUnitTest is Test {
         });
 
         vm.prank(rollupProcessor);
-        vm.expectRevert(ErrorLib.InvalidInputAmount.selector);
+        vm.expectRevert(ErrorLib.InvalidInputA.selector);
```

Comment:
With the removal of the check. This test don't make sense anymore. You are reverting because of 
```solidity
} else if (_auxData == 1) {
        if (_inputAssetA.assetType != AztecTypes.AztecAssetType.ERC20) {
            revert ErrorLib.InvalidInputA();
        }
```